FindYourFriends/screens/MapScreen.js

```diff
@@ -12,8 +12,47 @@ import Card from "../components/Card";
 import colors from "../constants/colors";
 import Input from "../components/Input";
 import facade from "../facade";
+import useMutation from "@apollo/react-hooks";
 
 const MapScreen = (props) => {
+
+  const [updatePosition, { loading, error, data, called }] = useMutation(
+    facade.UPDATE_POSITION
+  );
```

Comment:
You sure this won't just fire immediately when the codes reaches here?

Reply:
Why would it? It's not a `query`

Reply:
Ok. I thought everything that didn't have "lazy" would fire immediately on Component Mount.